articles/discipline-flow.md

```diff
@@ -0,0 +1,62 @@
+## The Basic Model
+The very first step of any kind of meaningful product development must take the user as well as the business goals into account and by this means allow for continuous delivery of product increments. Relations between the Business (BUS), Customer Experience (CX), User Experience (UX), Requirements (REQ), Architecture (ARCH), Implementation (IMPL) and Operations (OPS) define the basic flow model to achieve this.  
+
+![Discipline Flow](./discipline-flow/dflow-sketch.jpg)
+
+The disciplines work on discipline-specific essentials: 
+
+- BUS: Business, organisational and societal goals for the given product(s)
+- CX: Customer's overall journey
+- UX: User and his goals
+- REQ: Product features and acceptance criteria
+- ARCH: Quality attributes and architecture
+- IMPL: Components and connectors
+- OPS: Infrastructure and monitoring
+
+Additionally, one might think of Testing (TST) as a discipline on its own, for instance between IMPL and OPS or OPS and the delivered Product. In some project contexts this would make perfectly sense, however in general we'd more like to think of testing as an integrated activity within each discipline. For instance in BUS we may think about how to test the business model hypothesis, as part of UX we may think about how to do the User Acceptance testing and in ARCH the testing for Quality Attributes is paramount. Of course, all this needs general testing knowledge as a foundation and this is where Testing as a discipline might play an explicit role in your project again.
+
+The different disciplines are grouped into three major parts, each with a different focus on what has to be achieved: (1) Understand what is needed (2) Develop what is needed (3) Deliver what is needed. These three quite different perspectives also provide hints on characteristics and attributes needed in your team.
+
+We then monitor the performance and usage of what was delivered, incorporate feedbacks and start again to develop corresponding improvements and extensions. 
+
+
+## Discipline Experts 
+The disciplines and relations described above form the core elements of an abstract product delivery machinery. However, the machinery is only the basic structure. It needs to be implemented by people who are experts in their disciplines. At the same time they must have general knowledge of all other disciplines in order to be able to put specific work always into context of the overall product lifecycle. Only then we may avoid silos and waterfall-like delivery processes.
+
+The more direct the disciplines are related, the more this knowledge is required. For instance working on the architecture requires a solid understanding of all the requirements, both the functional requirements as well as the requirements for quality attributes. And in the downstream flow we need a thourough understanding which infrastructure is needed for effective operations. This holds not only for production but also for development and test environments.
```

Comment:
Was wollt ihr mit diesem Absatz sagen?
Man muss alles kennen? Können? Wissen?

Reply:
Man sollte Experte in zumindest einer Disziplin sein und von den anderen soviel wissen dass man die eigene Arbeit immer auch im Kontext der anderen Disziplinen versteht

Reply:
Soll ich das nochmal umschreiben? In welcher Richtung denkst Du fehlt hier noch etwas, bzw. ist vielleicht auch etwas zuviel/überflüssig geschrieben?

Reply:
"The more direct the disciplines are related, "
Sind nicht immer alle Disciplines am Start in komplexen Softare Produkt Initiativen?

Reply:
"And in the downstream flow we need a thourough understanding which infrastructure is needed for effective operations."
downstream flow? Was heisst das?
Mich verwirrt dieser Absatz, aber der deutsche Text oben macht Sinn.

"Downstream flow" tönt für mich nach Wasserfall und nicht nach "Value Discovery", "Schnell in die Produktion", "Testen und Experimentieren",...

Reply:
ah ok. Doch klar, i.d.R. sind alle Disziplinen immer dabei, ich versuch das umzuschreiben